File: BackEnd/Models/Funcionario.cs
using System.ComponentModel.DataAnnotations;

public class Funcionario
{
    [Key]
    public int Id { get; set; }
    public string? Nome { get; set; }
    public string? Cargo { get; set; }
    public Aeroporto? Aeroporto { get; set; }
    public int AeroportoId { get; set; }
}

File: FrontEnd/AppPassagem/src/app/models/Funcionario.ts
import { Aeroporto } from "./Aeroporto";

export class Funcionario {
    id : number = 0;
    nome : string = "";
    cargo : string = "";
    aeroporto : Aeroporto;
    aeroportoId: number = 0;

    constructor(nome: string, cargo: string, aeroporto: Aeroporto){
        this.nome = nome
        this.cargo = cargo
        this.aeroporto = aeroporto
    }
}